variables.tf

```diff
@@ -3,6 +3,7 @@
 # -------------------------------------------------------------------------------------------------
 variable "vpc_cidr" {
   description = "The VPC CIDR to use for this VPC."
+  type        = string
```

Comment:
```suggestion ?
  type        = list(string) 
```

Reply:
It is wrong. Check here 
https://github.com/terraform-aws-modules/terraform-aws-vpc?ref=v3.11.0#input_cidr

Reply:
https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/variables.tf#L15